tests/run-make/c-link-to-rust-staticlib/rmake.rs

```diff
@@ -0,0 +1,22 @@
+// This test checks that C linking with Rust does not encounter any errors, with a static library.
+// See https://github.com/rust-lang/rust/issues/10434
+
+//@ ignore-cross-compile
+//@ ignore-freebsd
```

Comment:
Is there a good reason for `ignore-freebsd`?

Reply:
I found [this commit](https://github.com/rust-lang/rust/commit/e9108cd7b80ad3ff3f2c1f93ad2807876596d758) which added the FreeBSD ignore. Seems linked to #12460, which was a looooong time ago and had a different-looking Makefile back then. Let's see if we can remove it.

You can see how cursed this really is, because here, they fix a test on FreeBSD... [by adding one second to the sleep command](https://github.com/rust-lang/rust/pull/13288/commits/0d9fd8e2a1f8aa43b4cf66272eaa3cf695bbdcd9).